content/applications/hr/fleet/configuration.rst

```diff
@@ -0,0 +1,234 @@
+| Q5    | Serie Z4     | Class M   | Meriva        |
++-------+--------------+-----------+---------------+
+| TT    |              | Class R   | Mokka         |
++-------+--------------+-----------+---------------+
+|       |              | Class S   | Zafira        |
++-------+--------------+-----------+---------------+
+|       |              | Class SLK | Zafira Tourer |
++-------+--------------+-----------+---------------+
+|       |              | Class SLS |               |
++-------+--------------+-----------+---------------+
+
+.. _configuration/add-model:
+
+Add a new model
+---------------
+
+New vehicle models can easily be added to the database. To add a new model, navigate to
+:menuselection:`Fleet app --> Configuration --> Vehicle Models`. Click :guilabel:`Create` and a
+vehicle model form will load. Enter the following information on the form, then click
+:guilabel:`Save`. Be advised, some fields are specific to Belgian based companies, so not all fields
+or sections may be visible depending on the location of the company.
+
+- :guilabel:`Model name`: Enter the model name in the field.
+- :guilabel:`Manufacturer`: Select the manufacturer from the drop-down menu. If the manufacturer is
+  not configured, type in the manufacturer and then click :guilabel:`Create` or :guilabel:`Create
+  & Edit`.
+- :guilabel:`Vehicle Type`: Select one of two preconfigured vehicle types, either :guilabel:`Car` or
+  :guilabel:`Bike`, from the drop-down menu. The vehicle types are hardcoded in Odoo and are
+  integrated with the *Payroll* application. Adding additional vehicle types is not possible as it
+  will affect payroll.
+- :guilabel:`Category`: Select the category the vehicle is categorized under from the drop-down
+  menu. To create a new category, type in the category and then click :guilabel:`Create
+  (new category)`.
+
+.. note::
+   When the manufacturer is selected, the logo for the manufacturer will automatically load in the
+   image box in the top right corner.
+
+Information tab
+~~~~~~~~~~~~~~~
+
+Model
+*****
+
+- :guilabel:`Seats Number`: Enter how many passengers the vehicle can accommodate.
+- :guilabel:`Doors Number`: Enter the number of doors the vehicle has.
+- :guilabel:`Color`: Enter the color of the vehicle.
+- :guilabel:`Model Year`: Enter the year the vehicle was manufactured.
+- :guilabel:`Trailer Hitch`: Check this box if the vehicle has a trailer hitch.
+
+Salary
+******
+
+- :guilabel:`Can be requested`: Check this box if employees can request this model vehicle.
+- :guilabel:`Catalog Value (VAT Incl.)`: Enter the :abbr:`MSRP (Manufacturer's Suggested Retail
+  Price)` for the vehicle at the time of purchase or lease.
+- :guilabel:`C02 fee`: This value is specific to Belgian based companies only. If logged into a
+  Belgian company, this value is automatically calculated based on Belgian laws and regulations, and
+  cannot be modified. The value is based on the value entered in the :guilabel:`CO2 Emissions` field
```

Comment:
any other European countries?

Reply:
let's test this out or ask an expert-- talking about Belgian based companies so much in your docs makes the software seem like it is just suited for that market, we are worldwide-- and can accommodate lots of different situations. Also this is best suited for a note block or something like that--- important

Reply:
I did- I asked Quentin abut all of these fields because there was NO way I was able to figure that out myself! =D 

There are unfortunately a bunch of fields that are Belgian-specific or only appear in a Belgian localization. I notice that is the case for a bunch of areas in our HR docs.

Reply:
I checked again- and the section doesn't even appear for non-Belgian companies So I updated this with an intro sentence.